src/components/Navbar/Navbar.js

```diff
@@ -1,15 +1,16 @@
 import "../Navbar/Navbar.css";
+import {Link} from "react-router-dom";
 
 export default function Navbar(){
     return(
         <header className="header">
         <div className="lilac-navbar">
             {/* logo */}
```

Comment:
remove commented code

Reply:
Sure ,I will remove .
Thanks for reviewing.